src/gromacs/nblib/CMakeLists.txt

```diff
@@ -38,13 +38,47 @@
 # \author Sebastian Keller <keller@cscs.ch>
 #
 
-file(GLOB NBLIB_SOURCES *.cpp)
+add_library(nblib "")
```

Comment:
Historically, an empty sources list in the `add_library` call is problematic in some build systems. Check the CMake docs and try the environments described there. This might be fine by now...

Reply:
My understanding is that this is now the canonical way to do this.

Reply:
Ah. I think we have CMake 3.11 to thank for that.